src/lwc/displayRecords/displayRecords.js

```diff
@@ -2,20 +2,43 @@ import { LightningElement, track } from 'lwc';
 import getContact from '@salesforce/apex/FetchingRecords.getContact';
 import getOpportunity from '@salesforce/apex/FetchingRecords.getOpportunity';
 import getCases from '@salesforce/apex/FetchingRecords.getCases';
+import getAccountList from '@salesforce/apex/FetchingRecords.getAccountList';
 
 export default class DisplayRecords extends LightningElement {
     listContact;
     listOpp;
     listCase;
-    idAccount;
+    @track nameAccount;
+    @track data =[];
+    @track itemList=[];
+    @track value;
+    view=false;
+    viewTable=false;
+    @track idAccount;
+    column=[{fieldName:'Name', label:'Name'},{fieldName:'Id', label:'Id'}];
+    caseColumn=[{fieldName:'Id', label:'Id'},{fieldName:'CaseNumber', label:'Case Number'}];
     handleClick(event){
+        this.view=true;
         var inp = this.template.querySelector("lightning-input");
-        this.idAccount = inp.value;
-        //this.idAccount=this.template.querySelector('lightning-input');
+        this.nameAccount = inp.value;
+        getAccountList({input : this.nameAccount})
+        .then(result => {
+            let listAccounts = result;
+            this.itemList=[];
+            for(var i=0;i<listAccounts.length;i++) {
+                this.itemList.push({label: listAccounts[i].Name, value: listAccounts[i].Id});
+            }
+            this.data = this.itemList;
+        });
+    }
+    get optionsAccount() {
+        return this.data;    
+    }
+    handleChange(event){
+        this.idAccount=event.detail.value;
+        this.viewTable=true;
```

Comment:
This should be moved into the .then part of the "getContact()" so tables will be visible only after getting the result from Apex.